.ci/scripts/install-tools.bat

```diff
@@ -12,5 +12,5 @@ IF ERRORLEVEL 1 (
  curl -sL -o %WORKSPACE%\bin\gvm.exe https://github.com/andrewkroh/gvm/releases/download/v0.2.1/gvm-windows-amd64.exe
 )
 FOR /f "tokens=*" %%i IN ('"gvm.exe" use %GO_VERSION% --format=batch') DO %%i
-
+set GO111MODULE=off
```

Comment:
FYI: There is no need to set `GO111MODULE` to `off`. Just change the next line to `go install -mod=vendor github.com/magefile/mage`.